context-free.md

```diff
@@ -2,21 +2,54 @@
 
 ##  Who is this programming language for?
 
+This programming language is for anyone who wants to create images using written instructions.
```

Comment:
That's a good point - it goes beyond written instructions and towards what type of written instructions are used.